<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>643</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>600</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionFile_Select"/>
     <addaction name="actionFolder_Select"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuBlur_median">
     <property name="title">
      <string>Blur</string>
     </property>
     <addaction name="actionMedian"/>
     <addaction name="actionGaussian"/>
    </widget>
    <widget class="QMenu" name="menushift">
     <property name="title">
      <string>shift</string>
     </property>
     <addaction name="actionEast"/>
     <addaction name="actionWest"/>
     <addaction name="actionSouth"/>
     <addaction name="actionNorth"/>
    </widget>
    <addaction name="actionto_Gray"/>
    <addaction name="actionCrop"/>
    <addaction name="menuBlur_median"/>
    <addaction name="menushift"/>
    <addaction name="actionRotate"/>
    <addaction name="actionNext"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFile_Select">
   <property name="text">
    <string>File Select</string>
   </property>
  </action>
  <action name="actionFolder_Select">
   <property name="text">
    <string>Folder Select</string>
   </property>
  </action>
  <action name="actionto_Gray">
   <property name="text">
    <string>to Gray</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionBlur_Gaussian">
   <property name="text">
    <string>Blur(Gaussian)</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>median</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>gaussian</string>
   </property>
  </action>
  <action name="actionEast">
   <property name="text">
    <string>east</string>
   </property>
  </action>
  <action name="actionWest">
   <property name="text">
    <string>west</string>
   </property>
  </action>
  <action name="actionSouth">
   <property name="text">
    <string>south</string>
   </property>
  </action>
  <action name="actionNorth">
   <property name="text">
    <string>north</string>
   </property>
  </action>
  <action name="action_90">
   <property name="text">
    <string>+90</string>
   </property>
  </action>
  <action name="action_91">
   <property name="text">
    <string>-90</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>rotate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
